src/Swarm/TUI/Controller.hs

```diff
@@ -719,7 +719,38 @@ resetREPL t r ui =
 
 -- | Handle a user input event for the REPL.
 handleREPLEvent :: BrickEvent Name AppEvent -> EventM Name AppState ()
-handleREPLEvent = \case
+handleREPLEvent x = do
+  s <- get
+  let controlMode = s ^. uiState . uiREPL . replControlMode
+  case x of
+    ControlKey 'd' -> do
```

Comment:
We already use Control D for quit (when the REPL is empty) to make it easier for players coming from shell. How about Meta D?

Reply:
I've renamed the mode from "driving" to "piloting".  Changed the mode switch key to `Meta-p`.  Added other commands like grab, harvest.  Now is only available via Creative mode:

1. Run in `cheat mode`
    stack run -- --scenario data/scenarios/Tutorials/move.yaml --cheat
2. Then type Ctrl+v to enter creative mode
3. Then type Meta+p to enter piloting mode